### PR TITLE
docs for user data deletion

### PR DESCRIPTION
**Pull request by ritvik-statsig**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://62d6dc7f0028f94c2e9a5fef--cozy-fox-0defec.netlify.app

File: docs/user_data_deletion_requests.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## Sending Requests
+
+Data deletion requests take the following parameters:
+- `unit_type`: The unit type that corresponds to the IDs that you would like to deleted. This could be userID, stableID, or some custom ID that you have set up
+- `ids`: A list of strings, that are the IDs you would like to delete data for
+- `request_id` (optional): If you store a request ID on your side that you would like us to use, you can pass it in, and it will be used to track the request. If left unset, we will provide an ID for you to use in the response. Note that it must be a unique ID if you are supplying your own, otherwise the request will fail with a 400 code.
+
+```bash
+curl \
+  --header "statsig-api-key: <YOUR-API-KEY>" \
```
Comment (tore-statsig) [inline docs/user_data_deletion_requests.md:24]: This should probably be under console API, right?  And use a console api key?  This doesnt spell out which api key to use here.

cc @jacob-statsig  RE console api

Comment (tore-statsig) [reply docs/user_data_deletion_requests.md:24]: Whoops just reread and saw that it does require a console api key.  This should be documented under `docs/console-api/` then

Comment (ritvik-statsig) [reply docs/user_data_deletion_requests.md:24]: cc @marcos-statsig for what you think about APIs like this and mark_data_ready which are sort of console APIs but also not really console APIs. assuming we will eventually have some docs for that one too right?

File: docs/user_data_deletion_requests.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### How to use the API
+
+All requests must include the STATSIG-API-KEY field in the header. The value should be a Console API Key which can be created in the Project Settings on console.statsig.com/api_keys.
+
+## Sending Requests
+
+Data deletion requests take the following parameters:
+- `unit_type`: The unit type that corresponds to the IDs that you would like to deleted. This could be userID, stableID, or some custom ID that you have set up
+- `ids`: A list of strings, that are the IDs you would like to delete data for
```
Comment (tim-statsig) [inline docs/user_data_deletion_requests.md:19]: Do we want to update this to comma deliminated?

Comment (ritvik-statsig) [reply docs/user_data_deletion_requests.md]: updated

**Pull request merged**